.github/workflows/daily-e2e.yml

```diff
@@ -0,0 +1,44 @@
+name: 'Daily E2E Tests'
+
+on:
+   pull_request:
+   schedule:
+     - cron:  '0 0 * * *'
+
+jobs:
+    e2e-tests:
+        runs-on: ubuntu-18.04
+        strategy:
+            fail-fast: false
+            matrix:
+                wordpress: ['https://wordpress.org/latest.zip', 'https://wordpress.org/nightly-builds/wordpress-latest.zip']
+                woocommerce: ['https://downloads.wordpress.org/plugin/woocommerce.zip', 'https://github.com/woocommerce/woocommerce/releases/download/nightly/woocommerce-trunk-nightly.zip']
+                exclude: 
+                    - {'wordpress': 'https://wordpress.org/nightly-builds/wordpress-latest.zip', 'woocommerce': 'https://github.com/woocommerce/woocommerce/releases/download/nightly/woocommerce-trunk-nightly.zip'}
+        steps:
+            - name: Check out repository code
+              uses: actions/checkout@v2
+            - name: Install PHP dependencies
+              run: |
+                  composer install  --no-dev
+            - name: Setup Node.js
+              uses: actions/setup-node@v2-beta
+              with:
+                  node-version: '14'
+            - name: Build
+              run: |
+                  npm i
+                  composer require wp-cli/i18n-command
+                  npm run build:feature-config
+                  npm run build
+            - name: Setup wp-env
+              env:
+                  WP_ENV_CONFIG: '{ core: "${{ matrix.wordpress }}", plugins: [ ".", "${{ matrix.woocommerce }}" ] }'
+              run: |
+                  npm -g i @wordpress/env
+                  printf '%s\n' "$WP_ENV_CONFIG" > .wp-env-override.json
+                  WP_ENV_TESTS_PORT=8084 wp-env start
+                  wp-env run tests-cli "wp post create --post_type=page --post_status=publish --post_title='Ready' --post_content='E2E-tests.'"
+            - name: Test
+              run: |
+                  npm run test:e2e
```

Comment:
Just thought i'd mention, I'm doing quite a large refactor of e2e and one thing I've done is remove these package json commands in favor of calling wc-e2e directly.

There are a couple of things you could cherry pick from my branch if you want to do that here (since I don't know who will get merged first).

1. this command should be `npx wc-e2e test:e2e`
2. You'll need my changes to process ts files which are here: https://github.com/woocommerce/woocommerce-admin/pull/6682/files#diff-4c2f296d817ca80b3f5fb4fea1175116daa54294e6a9252251acfe1b02f9de17R10

Reply:
Ah thanks, I was wondering whether the `npx` command should be used here but was getting the TS related errors.

I've the changed the command in d5bf4f9. Let's wait until your refactor is merged and then I'll refresh this to fix the errors.

Reply:
The refactor is now merged and I've refreshed this. Tests are passing 👍